new-packages/ts-project/src/index.ts

```diff
@@ -183,9 +55,13 @@ export function createProject(
       if (!sourceFile) {
         return [];
       }
-      return findCreateMachineCalls(ts, sourceFile).map((call) =>
-        extractMachineConfig(ts, call),
-      );
+      return findCreateMachineCalls(ts, sourceFile).map((call) => {
+        const ctx: ExtractionContext = {
+          errors: [],
+          sourceFile,
```

Comment:
it's not apparent why this has been added here since it stays unused

Reply:
Was added for a code that's already deleted. I'll delete it.